scipy/special/tests/test_mpmath.py

```diff
@@ -1177,6 +1177,32 @@ def test_erfi_complex(self):
                             mpmath.erfi,
                             [ComplexArg()], n=200)
 
+    def test_ndtr(self):
+        assert_mpmath_equal(sc.ndtr,
+                            _exception_to_nan(lambda z: mpmath.erfc(-mpmath.mpmathify(z/np.sqrt(2.)))/2.),
+                            [Arg()], n=200)
+
+    def test_ndtr_complex(self):
+        assert_mpmath_equal(sc.ndtr,
+                            lambda z: mpmath.erfc(-z/np.sqrt(2.))/2.,
+                            [ComplexArg(a=complex(-10000, -10000), b=complex(10000, 10000))], n=200)
+
+    def test_log_ndtr(self):
+        # limit the range of z > 0: in this regime log_ndtr(z) = log(1 - exponentially_small_correct),
```

Comment:
Great idea, implemented in the last installment. This pushes the underflow from x=6 to more than 45, which should be enough for everybody (tm).